packages/core/integration-tests/test/integration/scope-hoisting/commonjs/mutated-non-static-require/index.js

```diff
@@ -0,0 +1,4 @@
+let x = require('./other');
+x['fo' + 'o'] = 4;
```

Comment:
Doesn't swc simplify this so that it's just `x.foo`? (Not sure if it makes a difference for the test though)

Reply:
Hmm appears not. I looked at the output and it had:

```js
$1d3bc13f8d088d40038ce7804a219fa1$exports['fo' + 'o'] = 4;
```

Not sure why SWC doesn't optimize it.